jsonargparse_tests/test_deprecated.py

```diff
@@ -155,6 +155,14 @@ def test_logger_property_none(self):
             LoggerProperty(logger=None)
 
 
+    @unittest.mock.patch.dict(os.environ, {"JSONARGPARSE_DEPRECATION_WARNINGS": "all"})
```

Comment:
Remove this. The only test in which `env_prefix=None` should be this one, so only here we would see this warning and there is no need to enable all warnings.

Reply:
I noticed that I needed this when running the tests in my local venv. Seems ok now